swatch-picker/open-swatch-picker.js

```diff
@@ -0,0 +1,2204 @@
+
+
+const styles = new CSSStyleSheet()
+styles.replaceSync(`
```

Comment:
```suggestion
styles.replaceSync(/* css */`
```

This should make this code syntax highlight as CSS in some editors.